pymongo/collection.py

```diff
@@ -1290,6 +1290,16 @@ def find(self, *args, **kwargs):
            expression object.
            Soft deprecated the ``manipulate`` option.
 
+        .. versionchanged:: 4.0
```

Comment:
Please merge this section with the existing `versionchanged:: 4.0` above.

Reply:
Done.